.mci.yml

```
@@ -12,7 +12,8 @@ variables:
     # If updating mongoc_version_minimum, also update:
     # - the default value of --c-driver-build-ref in etc/make_release.py
     # - LIBMONGOC_REQUIRED_VERSION in src/mongocxx/CMakeLists.txt
-    mongoc_version_minimum: &mongoc_version_minimum "1.25.0"
+    # - the PURL version in etc/purls.txt
+    mongoc_version_minimum: &mongoc_version_minimum "8a85033"
```

```suggestion
    mongoc_version_minimum: &mongoc_version_minimum "1.28.0"
```

Update to 1.28.0 now that C driver is released.